cfgov/v1/templatetags/email_popup.py

```diff
@@ -0,0 +1,26 @@
+from django import template
+from django.conf import settings
+from django.template.loader import render_to_string
+from django.utils.safestring import mark_safe
+
+from flags.state import flag_enabled
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def email_popup(request):
```

Comment:
This is interesting --- I'm not crazy about using context functions but it does provide a cleaner way for handling lists of URLs.